frmts/netcdf/netcdfdataset.cpp

```diff
@@ -4777,6 +4777,49 @@ void netCDFDataset::SetProjectionFromVar( int nGroupId, int nVarId,
     if( !bGotGeogCS && !bGotCfSRS && !bGotGdalSRS && !bGotCfGT && !bGotCfWktSRS)
         CPLDebug("GDAL_netCDF", "did not get projection from CF nor GDAL!");
 
+    // wish of 6195
+    // we don't have CS/SRS, but we do have GT, and we live in -180,360 -90,90
+    if (!bGotGeogCS && !bGotCfSRS && !bGotGdalSRS && !bGotCfWktSRS)  
+    {
+      if (bGotCfGT || bGotGdalGT) 
+      {
+        bool bAssumedLongLat = false; 
+        // check setting of GDAL_NETCDF_ASSUME_LONGLAT config option.
+        const char *pszCfgValue = CPLGetConfigOption("GDAL_NETCDF_ASSUME_LONGLAT", nullptr);
```

Comment:
it could perhaps be valuable to have this as an option option as well (or in replacement). We tend to avoid configuration options as they have a global effect. The advantages of open options is that they can be set in a VRT for example. Look for ``CSLFetchNameValueDef(papszOpenOptions, "IGNORE_XY_AXIS_NAME_CHECKS"`` in this file for an example.

Reply:
I'll look into the open option, I definitely want this as global though. This will make a lot of sources "just work"when read through the warper, without the need for vrt or any other curation, which is my primary motivation.

Reply:
it can be both, like IGNORE_XY_AXIS_NAME_CHECKS which is an open option and GDAL_NETCDF_IGNORE_XY_AXIS_NAME_CHECKS a config option. Other benefit of open options is that when you open a raster file in QGIS, they are proposed in the UI (in "Open raster file" dialog), since they are documented in the GDAL_DMD_OPENOPTIONLIST XML snippet (see around line 10569)

Reply:
thank you, understood - commited and pushed now -  the behaviour seems right and I've baked it into the py tests